### DISCUSSION TITLE
Server-side Translations with no-prefix

### DISCUSSION THREAD

**Discussion by KarelVendla** [nuxt-modules/i18n#2667] [Ideas]
### Describe the feature

Currently it seems Server-Side translating does not work with no_prefix strategy.
Could this be a feature for the future?

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by kazupon** [comment]
Thank you for your feedback.

Currently, server-side translations is experimental feature.
So, It does not integrate with locale detect by nuxt i18n routing strategy.
You need to implement a locale detector. As described in the documentation.
https://i18n.nuxtjs.org/guide/server-side-translations#define-locale-detector

If you have an idea, we are waiting for your it in github discussions.

Thanks!
